ci/builder/ssh_known_hosts

```diff
@@ -1,4 +1,6 @@
 mtrlz.dev ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsJdMnBVoCu1StAcoSH9iMIdM94V3if6U87gdrucDV/NHeWSTZa3lxHV4mGAFfSBV0oF3y82Sx4gspTjBxAeqwLcm70DrtSzl4cEPiJo4Hgj9U54kuZcIpDP8He/rNYzncrzKoeKzOwtVH5+oaJPcfeZ2CI1BVGj/Xt2A44d8Bixtzc6r+Q9czzYIwTCSxN4+rr7wPnNT3QXF+EvmmIsChXfsjVJpFHQKygVd9+9hk+b/bnqjsMG0O2gHmXSh4FQgbjeJvdTLEqC6SN8VEkaLLosbn79gxjei+U8pHcnossNgJQiaj4N5MDQwzWwGV9dxFBvvxcpdUknIBFlML5Joj
 mtrlz.dev ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMl4Vm8V+nUA677qNBKcIjUACY1tZuUa74MVMScGUp+szeuPbkqHpYqSkGnU+ONYVT0nRmXFhIOU2cO6hi2lGs4=
 mtrlz.dev ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPbUN6RkJgvRv3Y7pmnwth5nribjue21mf8BeD12qr2C
-github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
+github.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOMqqnkVzrm0SdG6UOoqKLsabgH5C9okWi0dh2l9GKJl
+github.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCj7ndNxQowgcQnjshcLrqPEiiphnt+VTTvDP6mHBL9j1aNUkY4Ue1gvwnGLVlOhGeYrnZaMgRK6+PKCUXaDbC7qtbW8gIkhL7aGCsOr/C56SJMy/BCZfxd1nWzAOxSDPgVsmerOBYfNqltV9/hWCqBywINIR+5dIg6JTJ72pcEpEjcYgXkE2YEFXV1JHnsKgbLWNlhScqb2UmyRkQyytRLtL+38TGxkxCflmO+5Z8CSSNY7GidjMIZ7Q4zMjA2n1nGrlTDkzwDCsw+wqFPGQA179cnfGWOWRVruj16z6XyvxvjJwbz0wQZ75XK5tKSb7FNyeIEs4TT4jk+S4dhPeAUC5y+bDYirYgM4GC7uEnztnZyaVWQ7B381AK4Qdrwt51ZqExKbQpTUNn+EjqoTwvqNj4kqx5QUCI0ThS/YkOxJCXmPUWZbhjpCg56i+2aB6CmK2JGhn57K5mj0MNdBXA4/WnwH6XoPWJzK5Nyu2zB3nAZp+S5hpQs+p1vN1/wsjk=
```

Comment:
Looks different than what's listed publicly, but could be an encoding issue. https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints

Reply:
Never mind, looks good! We might want to add the ECDSA key, too.

Reply:
ecdsa is in line 6